src/govuk/components/exit-this-page/_index.scss

```diff
@@ -78,4 +78,8 @@
     left: 0;
     background-color: govuk-colour("white");
   }
+
+  .govuk-exit-this-page-hide-content *:not(.govuk-exit-this-page__overlay) {
```

Comment:
`:not` won't work in IE8, but I'm not gonna lose sleep over it.

Reply:
I'm not either _but_ I've scooped it out for now until we move to v5. Good spot.